### DISCUSSION TITLE
version error

### DISCUSSION THREAD

**Discussion by Solmyribn** [Digitalroot-Valheim/Digitalroot.Valheim.ForsakenJVL#2] [Q&A] [answered]
hello, I have a version error for Forsaken
I tested the "799-0-7-0-1618198629" and the "799-0-6-0-1618033834"

I have this : [Error  :   BepInEx] Could not load [Digitalroot ForsakenJVL 1.1.0] because it is incompatible with: com.bepinex.plugins.forsaken

Can you tell me which version you used.

THX

**Comment by Digitalroot** [comment, marked as answer]
> Thx Discord Odin Plus Team
> the problem comes from a conflict between the 2 dll
> Delete forsakenmod.dll in \steamapps\common\Valheim\BepInEx\plugins
> but keep the "forsakenmod" well.
#3

**Comment by HammerZword** [comment]
[Error  :Digitalroot] Unable to find asset file 'forsakenmod', please make sure 'forsakenmod' and 'Digitalroot.Valheim.ForsakenJVL.dll' are in C:\Users\korie\AppData\Roaming\r2modmanPlus-local\Valheim\profiles\Default\BepInEx\plugins\Digitalroot.Valheim.ForsakenJVL
[Error  :Digitalroot] Digitalroot.Valheim.ForsakenJVL is not loaded
[](url)
.
![al](https://user-images.githubusercontent.com/97345805/148645555-8c7c1707-d6b1-403c-ad25-9ae90da019ad.png)
How do i fix this :)

**Reply by Digitalroot** [reply]
There is an issue with the mod and R2. I'll post a fix for it soon.
